rust/processor/src/models/fungible_asset_models/v2_fungible_asset_utils.rs

```diff
@@ -187,6 +187,76 @@ impl FungibleAssetSupply {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ConcurrentFungibleAssetSupply {
+    pub current: AggregatorU128,
+}
+
+impl ConcurrentFungibleAssetSupply {
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Thanks for the observation. I am now pre-processing and storing the ConcurrentFungibleAssetSupply in object_metadatas. I modified FungibleAssetMetadataModel to store the ConcurrentFungibleAssetSupply in supply_v2 and maximum_v2.